lib/urbanopt/geojson/schema/building_properties.json

```diff
@@ -225,6 +225,45 @@
     },
     "user_data": {
       "description": "Arbitrary user data"
+    },
+    "ev_charging": {
+      "description": "Should be set to true if EV charging is associated with the building.",
+      "type": "boolean"
+    },
+    "ev_charging_station_type": {
+      "description": "Indicates whether EV charging associated with the building is for Typical Home, Public or Work charging station",
+      "type": "string",
+      "enum": [
+        "Typical Home",
+        "Typical Public",
+        "Typical Work"
+      ]
+    },
+    "delay_type": {
+      "description": "Adds workplace charging flexibility for different scenarios. Min delay scenario indicates EV charging immediately upon arriving to work, Max delay indicates not charging until necessary thereby shifting EV charging load to later in the day and Min power indicates charging EVs at a minimum rate over the parking event.",
+      "type": "string",
+      "enum": [
+        "Min Delay",
+        "Max Delay",
+        "Min Power"
+      ]
+    },
+    "ev_charging_behavior": {
+      "description": "Describes scenarios for EV charging behavior, Business as Usual implies home dominant charging behavior, Free Workplace Charging at Project Site implies peak power draw from EV charging during morning hours due to EV charging at workplaces and Free Workplace Charging Across Metro Area scenario reduces Home EV charging relative to Free Workplace Charging at Project Site for residents who work elsewhere and can charge their vehicles for free at those workplaces.",
+      "type": "string",
+      "enum": [
+        "Business as Usual",
+        "Free Workplace Charging at Project Site",
+        "Free Workplace Charging Across Metro Area"
+      ]
+    },
+    "ev_percent": {
+      "description": "Denotes the % of vehicles parked at the building that are EVs.",
+      "type": "number"
+    },
+    "ev_curtailment_frac": {
```

Comment:
For my education, why is this a fraction and not a percentage like the one above? Feels a little strange that they're not using the same units. LGTM 👍

Reply:
These are two different measures, could be because of that. Thanks for reviewing!